scope.md

```diff
@@ -19,3 +21,20 @@ hyper-specific methods for one type of data to general, do-it-all packages
 (e.g. matplotlib). pyOpenSci accepts packages that are somewhere in between the 
 two. If you're interested in submitting your data visualization package, please 
 open a pre-submission inquiry on first.
+
+## Python package technical scope
+
+pyOpenSci is still defining the technical scope criteria for package review.  
+Your package **may not be in technical scope** for us to review at this time if
+fits any of the out-of-technical-scope criteria listed below. 
+
+- Package is an out of sync fork of another package repository. 
+Sometimes we understand that a package maintainer may need to step down.  In that case, we strongly suggest that the original package owner, transfer the package to a new organization that would allow transfer of ownership of the package rather than 
+a fork. However, if there is a case where a forked repository is warranted, please 
+consider submitting a pre-submission inquiry first and explain why the package is a 
+fork rather than an independent parent repository.
+- **Vendored dependencies:** If your package is a wrapper that wraps a tool in another 
+language or a lower level tool, we prefer that the dependency be added as sa 
+dependency to your package so that maintenance of the original code base is independent from your package's maintenance. 
+- **If the code base of your package is exceedingly complex in terms of structure of maintenance needs**, we may not be able to review it. pyOpenSci has a goal of supporting long term maintenance of tools. When a maintainer steps down, we need to know that 
```

Comment:
Maybe this bullet point should be first--maybe it should even be more like an intro paragraph--and all the other ones should follow it? I think 1 and 2 are special cases of this one? What we really want is to avoid complex package structure, and forks of actively maintained packages, as well as packages that vendor a ton of code would both make things complex

Reply:
ok i've reorganized this and added those bullets as examples. and made the last one a IMPORTANT call out block.